packages/core/src/common/classes.ts

```diff
@@ -439,3 +440,20 @@ export function positionClass(position: Position | undefined) {
     }
     return `${NS}-position-${position}`;
 }
+
+export function sizeClass(
+    size: Size | Omit<Size, "small">,
```

Comment:
```suggestion
    size: Size,
```

This type is equivalent, a type of `Omit<Size, "small">` should be assignable to `Size`.

Reply:
bba8d5aec50ccfb1791117edf6bf63dee97a570c